crates/ruma-client-api/src/media/get_media_preview.rs

```diff
@@ -49,9 +50,9 @@ pub mod v3 {
     }
 
     impl Request {
-        /// Creates a new `Request` with the given url and timestamp.
-        pub fn new(url: String, ts: MilliSecondsSinceUnixEpoch) -> Self {
-            Self { url, ts }
+        /// Creates a new `Request` with the given url and optional timestamp.
```

Comment:
Mention of the timestamp should be removed from the docs

Reply:
Oops forgot about that too.